charts/sysdig-deploy/scripts/migrate_values.py

```diff
@@ -0,0 +1,67 @@
+import copy
+import sys
+import yaml
+
+
+## helpers for migrating values
+def populate_globals(new_values, values):
+    globals = {
+        "clusterConfig": {},
+        "sysdig": {},
+        "image": {},
+    }
+
+    name = values.pop("clusterName", "")
+    if name:
+        globals["clusterConfig"]["name"] = name
+
+    accessKey = values.get("sysdig", {}).pop("accessKey", "")
+    if accessKey:
+        globals["sysdig"]["accessKey"] = accessKey
+
+    accessKeySecret = values.get("sysdig", {}).pop("existingAccessKeySecret", "")
+    if accessKeySecret:
+        globals["sysdig"]["accessKeySecret"] = accessKeySecret
+
+    registry = values.get("image", {}).pop("registry", "")
+    if registry:
+        globals["image"]["registry"] = registry
+
+    new_values["global"] = globals
+
+
+def populate_agent(new_values, values):
```

Comment:
this function and the populate_node_analyzer feel low-key unintuitive with the copy-then-pop.

Reply:
fair point, explicitly naming the fields to copy probably reads better. will update with that